rpcd/playbooks/roles/rpc_maas/templates/client.my.cnf.j2

```diff
@@ -2,5 +2,5 @@
 
 [client]
 host={{ container_address }}
-user={{ galera_root_user }}
-password={{ galera_root_password }}
+user={{ galera_root_user|default('root') }}
+password={{ galera_root_password|default('secrete') }}
```

Comment:
Not that I have a dog in this fight, however I think it would be better for ansible to complain about an unset variable than it would be to assume it can be set to "secrete".